### PR TITLE
NEW Can select existing batch numbers for a combo list (in mrp consumtion)

### PR DESCRIPTION
**Pull request by SteiniGreenwork**
# New load existing batch numbers for item  within mrp consumtion
While consuming items within mrp, existing batch numbers for the corresponing item are loaded into select field

### CODE REVIEW

File: htdocs/mrp/mo_production.php [hunk 1]
```diff
@@ -967,7 +973,8 @@
 							print '<td>';
 							if ($tmpproduct->status_batch) {
 								$preselected = (GETPOSTISSET('batch-'.$line->id.'-'.$i) ? GETPOST('batch-'.$line->id.'-'.$i) : '');
-								print '<input type="text" class="width50" name="batch-'.$line->id.'-'.$i.'" value="'.$preselected.'">';
+								print '<td>'.$formproduct->selectLotStock('', 'batch-'.$line->id.'-'.$i, '', 1, 0, $line->fk_product, '', '', '', '', '', '', 1).'</td>';
```
Comment (eldy) [inline htdocs/mrp/mo_production.php:976]: It seems we are loosing the $preselected parameter.

Comment (SteiniGreenwork) [reply htdocs/mrp/mo_production.php:976]: Hi Eldy, i will try to add the new method like you sugessted but i need a few days because i'm on the road

Comment (SteiniGreenwork) [reply htdocs/mrp/mo_production.php]: PR Fixed

Comment (eldy) [reply htdocs/mrp/mo_production.php]: Can you paste a screenshot of the result ?

Comment (SteiniGreenwork) [reply htdocs/mrp/mo_production.php]: Bildschirmfoto vom 2021-10-04 19-39-44
Hi Eldy, attached the screenshot. I'm using Chrome  94.0.4606.71 on Fedora 34.
Using Chrome and Firefox on Windows also shows the dropdown with values. But it's a text field and the Browser accepts any value i entered. Best regards Stephan

**Pull request merged**